packages/service-library/tests/test_utils_secrets.py

```diff
@@ -65,3 +69,15 @@ def test_compare_secrets():
 def test_generate_token_secrets():
     secret_key = generate_token_secret_key()
     assert len(secret_key) == 2 * _MIN_SECRET_NUM_BYTES
+
+
+@pytest.mark.parametrize("start, end", [(1, 10), (99, 100)])
+async def test_secure_randint(start: int, end: int):
+    for _ in range(1000):
```

Comment:
MINOR: i guess there is no need to run this 1000 times now ... :-) Except if you intend to test builtin python library

Reply:
yeah. Let me refactor it